File: scripts/node/Player.cs
using System;
using UnityEngine;
[Serializable]
  public  class Player
    {
    public string name = "";
    public NodePosition position = new NodePosition();
    public NodeScale scale = new NodeScale();
    public PlayerColor color = new PlayerColor();
    public string id = "";
    public string score = "0";


    public Player()
    {

    }

    public Player (string ID, string Name, NodePosition pos, NodeScale s)
    {
        id = ID;
        name = Name;
        position = pos;
        scale = s;
    }

    public void AddScore (int v)
    {
        score = v.ToString();
    }
    }

File: Server/class/player.js
var shortID = require("shortid");
var vector2 = require("./vector2.js");
var vector3 = require("./vector3.js");
var color = require("./color.js");
module.exports = class player {
	constructor() {
		this.name = "";
		this.id = shortID.generate();
		this.position = new vector2();
		this.scale = new vector3();
		this.color = new color();
		this.score = 0;
		this.kills = 0;
	}
}